designcompose/src/main/java/com/android/designcompose/ComputePaths.kt

```diff
@@ -130,7 +131,9 @@ internal fun ViewShape.computePaths(
                 )
             }
             is ViewShape.Path -> {
-                getPaths(this.path, this.stroke)
+                // If strokePaths is not null, use it instead of this.stroke since it contains the
+                // stroked vector path of a progress vector.
+                getPaths(this.path, strokePaths ?: this.stroke)
```

Comment:
Is there a way to move this to the transform_flexbox instead of at runtime, or is it likely to regress other functionality?

Reply:
Do you mean saving the plugin data's stroked paths into the node's stroke list instead of separately?

Reply:
Done